src/app/pages/choose-account/components/accounts.tsx

```diff
@@ -85,7 +85,7 @@ function AddAccountAction() {
     <Box mx="space.05" py="space.02" onClick={onCreateAccount} {...bind}>
       <HStack alignItems="center">
         <PlusIcon />
-        Generate new account
+        <styled.span textStyle="label.02">Generate new account</styled.span>
       </HStack>
       {component}
```

Comment:
Shouldn't the container just be changed such that the height of the list stops when it gets to the button, and doesn't go under it?